### ISSUE TITLE
Load expect with chai/expect

### ISSUE THREAD

**Issue by llfm** [chaijs/chai#716]
Could `expect` have the same behavior of `should` [here](https://github.com/chaijs/chai/issues/594)?

The possibility to use:
- `import { expect } from 'chai'`
- `import expect from 'chai/expect'`

**Comment by meeber** [comment]
Hi @llfm.

With proper transpiling, this should already work:
```js
import { expect } from 'chai'
```

**Comment by keithamus** [comment]
@llfm see my comment [here](https://github.com/chaijs/chai/issues/594#issuecomment-173684778) for a complete list of available syntaxes (as of the upcoming `4.0.0` release):

```js
// Plugin-less
import { expect } from 'chai'
import { assert } from 'chai'
import 'chai/should'
// if you need should.exist/should.fail etc:
import should from 'chai/should'

// With plugins
import chai from 'chai';
import somePlugin from 'some-plugin';
const expect = chai.use(somePlugin).expect

import chai from 'chai';
import somePlugin from 'some-plugin';
const assert = chai.use(somePlugin).assert

import chai from 'chai';
import somePlugin from 'some-plugin';
const should = chai.use(somePlugin).should()
```
